joschahenningsen/TUM-Live
# Link to waiting room

### Motivation and Context
Currently, the new start page doesn't implement any waiting room logic whatsoever.

### Description
Add `isComingUp` to `StreamDTO`. Filter based on this boolean value and display accordingly.

### Steps for Testing
- 1 Course with upcoming stream

1. Log in
2. Navigate to a course page
3. UI of screenshot 1 should be visible
4. Navigate to `/new`
5. UI of screenshot 2 should visible 
6. ⭐️

### Screenshots
<img width="862" alt="image" src="https://github.com/joschahenningsen/TUM-Live/assets/29633518/8696eb11-6256-47d8-ba73-946b7eb9c307">

<img width="453" alt="image" src="https://github.com/joschahenningsen/TUM-Live/assets/29633518/8b6b38e2-769c-4ae0-aad0-8f8a66041371">
